Apps/WebClient/src/ClientApp/src/views/landing.vue

```diff
@@ -525,5 +544,12 @@ export default class LandingView extends Vue {
             }
         }
     }
+    .covid19-section {
+        background-color: #8bd5c7;
+
+        @media (max-width: 767px) {
+            margin-bottom: 10px !important;
+        }
```

Comment:
This custom CSS should be replaced with the Bootstrap classes `mb-3 mb-md-0`, which will apply a 1rem (16px) bottom margin for viewports below medium.

These classes could be placed on the \<b-row\>.  Also, the `m-0` that is there can probably be removed (since the horizontal margins are now being handled by `mx-n2`).